src/compiler/parser.ts

```diff
@@ -6814,6 +6817,69 @@ namespace ts {
                     return finishNode(result);
                 }
 
+                function parseAuthorTag(start: number, tagName: Identifier): JSDocAuthorTag {
+                    const result = <JSDocAuthorTag>createNode(SyntaxKind.JSDocAuthorTag, start);
+                    result.tagName = tagName;
+
+                    const comment = tryParse(() => tryParseAuthorNameAndEmail());
+
+                    if (comment) {

```

Comment:
you should probably call parseTagComments and add on whatever it finds in addition to the string returned by parseAuthorNameAndEmail.

Reply:
Done in 6e4f581